apps/examples/lcd_test/example_lcd.c

```diff
@@ -254,13 +254,105 @@ static void test_bit_map(void)
 	free(lcd_data);
 }
 
+static void test_fps(void)
+{
+	int fd_rtc = 0;
+	int fd_lcd = 0;
+	int p = 0;
+	char port[20] = { '\0' };
+	size_t len;
+
+	fd_rtc = open("/dev/rtc0", O_RDWR);
+	if (fd_rtc < 0) {
+		printf("ERROR: LCD FPS test, Fail to open rtc.\n");
+		return;
+	}
+
+	len = xres * yres * 2 + 1;
+	uint8_t *lcd_data_red = (uint8_t *)malloc(len);
+	uint8_t *lcd_data_blue = (uint8_t *)malloc(len);
+	if (lcd_data_red == NULL || lcd_data_blue == NULL) {
+		printf("FPS TEST, malloc failed for lcd data : %d\n", len);
+		close(fd_rtc);
+		return;
```

Comment:
This could cause memory leak.
For example, the `lcd_data_red` is allocated, but `lcd_data_blue` is failed. Then, this if conditional will return without freeing `red`. Same as opposite case.
Need to check both separately and to free preallocated memory before return.